DEVELOPMENT.md

```diff
@@ -44,6 +44,10 @@ generated code, only commit test files being updated and any updated cassettes.
 
 Run test with `DEBUG=true` to display api server calls being made during test execution. This can be useful during debug time.
 
+### Writing Tests
+
+When creating new entities/objects through the API as part of tests, make sure to assign result of `getUniqueEntityName()` to at least one field of this object (e.g. title or name). This ensures uniqueness as well as trackability - IOW if a CI run fails to clean up resources, we're able to track the entity to the CI run, as the unique name will contain CI run Id.
```

Comment:
```suggestion
When creating new entities/objects through the API as part of tests, make sure to assign result of `getUniqueEntityName()` to at least one field of this object (e.g. title or name). This ensures uniqueness as well as trackability - IOW if a CI run fails to clean up resources, we're able to track the entity to the CI run, as the unique name will contain CI run ID.
```

Reply:
`IOW` -> can we use basic language here?

Reply:
BASIC is outdated let's stick to english ;)